libs/vuepress/src/generators/application/files/common/__dot__vuepress/config.js.template

```diff
@@ -0,0 +1,64 @@
+module.exports = {
+  title: 'Vuepress Docs Boilerplate',
+  description: 'The tagline of my site',
+
+  /**
+   * Extra tags to be injected to the page HTML `<head>`
+   */
+  head: [
+    ['meta', { name: 'theme-color', content: '#3eaf7c' }],
+    ['meta', { name: 'apple-mobile-web-app-capable', content: 'yes' }],
+    ['meta', { name: 'apple-mobile-web-app-status-bar-style', content: 'black' }]
+  ],
+
+  theme: '@vuepress/theme-default',
+
+  /**
+   * Theme configuration, here is the default theme configuration for VuePress.
+   */
+  themeConfig: {
+    repo: '',
+    <% if (isVuepress2) { %>editLink:<% } else { %>editLinks:<% } %> false,
```

Comment:
Would it be easier to dup these files into both `vuepress-1` and `vuepress-2` so as to reduce the conditionals?